static/scripts/images.js

```diff
@@ -17,72 +17,86 @@ let currImage = 0;
 
 /**
  * Shows the modal image popup
- * 
+ *
  * @param {string} index index of the image in current page
  */
 function showImageModal(index) {
+	// Update the highlighting
+	for (let i = 0; i < pageImages.length; ++i) {
+		document
+			.getElementById(`modal-pagination-${i}`)
+			.classList.remove(`text-indigo-500`);
+	}
+	document
+		.getElementById(`modal-pagination-${index}`)
+		.classList.add(`text-indigo-500`);
 
-  // Update the highlighting
-  for (let i = 0; i < pageImages.length; ++i) {
-    document.getElementById(`modal-pagination-${i}`).classList.remove(`text-indigo-500`);
-  }
-  document.getElementById(`modal-pagination-${index}`).classList.add(`text-indigo-500`);
+	currImage = index;
+	const image = pageImages[currImage];
 
-  currImage = index;
-  const image = pageImages[currImage];
-
-  document.getElementById("modal-title").innerHTML = image.name;
-  document.getElementById("modal-excerpt").innerHTML = image.excerpt;
-  document.getElementById("modal-image").src = image.filepath;
-  document.getElementById("modal-text-date").innerHTML = image.date;
-  document.getElementById("modal-text-name").innerHTML = image.location.name;
-  modal.showModal();
+	document.getElementById("modal-title").innerHTML = image.name;
+	document.getElementById("modal-excerpt").innerHTML = image.excerpt;
+	document.getElementById("modal-image").src = image.filepath;
+	document.getElementById("modal-text-date").innerHTML = image.date;
+	document.getElementById("modal-text-name").innerHTML = image.location.name;
+	modal.showModal();
 }
 
 function nextImage() {
-  currImage = (currImage + 1) % pageImages.length;
-  showImageModal(currImage);
+	currImage = (currImage + 1) % pageImages.length;
+	showImageModal(currImage);
 }
 
 function prevImage() {
-  currImage = (currImage - 1 + pageImages.length) % pageImages.length;
-  showImageModal(currImage);
+	currImage = (currImage - 1 + pageImages.length) % pageImages.length;
+	showImageModal(currImage);
+}
+
+function navegateWithArrows() {
```

Comment:
The only useful change I can see here is this function - but you are not calling it anywhere, so the `keydown` events are never registered.

Could you rename this function to `setupArrowNavigation` and call it on the generated HTML? I suspect you might want to look at `images.templ`.

Please let me know if I've missed this, but I can't see how this is being setup.